app/javascript/react/components/Header/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Header = () => {
+  return (
+    <header>
+      <div className='container'>
```

Comment:
couldn't we just

```
<header className="container">
  <h1>Stats</h1>
</header>
```

note also that by this stage we should be thinking what a header may be an try to avoid any placeholder code - the placeholder was useful to prove we had react and graphql wired up but now that we have we should focus on working software no more placeholders